### PR TITLE
Update v-protocol README

### PR DESCRIPTION
**Pull request by jonathanl-bq**
This change updates the v-protocol README to indicate that we don't want vertica-nodejs users to look at it.

### CODE REVIEW

**Comment by jeremyprime** [comment]
Note that this README is used when publishing to npm (shows up on the website there), which is why it has information for the entire project and not just the individual package (so a user who finds one of the packages has an idea of how the entire project works).  Would it be better to just update the original README with a note about v-protocol not being meant to be used outside the nodejs driver?

I was thinking we could remove and gitignore the nested READMEs and just manage the README in the top-level directory.  Then update the publish script to copy the README to the package folders when publishing.  That way we can maintain a single README rather than modifying all 5 of them when something changes.

**Comment by jeremyprime** [comment]
> Note that this README is used when publishing to npm (shows up on the website there), which is why it has information for the entire project and not just the individual package (so a user who finds one of the packages has an idea of how the entire project works). Would it be better to just update the original README with a note about v-protocol not being meant to be used outside the nodejs driver?
> 
> I was thinking we could remove and gitignore the nested READMEs and just manage the README in the top-level directory. Then update the publish script to copy the README to the package folders when publishing. That way we can maintain a single README rather than modifying all 5 of them when something changes.

Sounds like we do want to move to maintaining individual READMEs per package, so ignore my comment.

File: packages/v-protocol/README.md [hunk 1]
```diff
@@ -4,65 +4,9 @@
...
-If using VS Code, you can install the `Remote - Containers` extension and it will use the configuration under the `.devcontainer` folder to automatically create dev containers, including Vertica.  See [here](https://code.visualstudio.com/docs/remote/containers) for more information on developing in containers using VS Code.  This process will complete steps 2 to 4 above.
-
-## Troubleshooting and FAQ
-
-The causes and solutions to common errors can be found among the [Frequently Asked Questions (FAQ)](https://github.com/vertica/vertica-nodejs/wiki/FAQ)
+Vertica protocol code used to send and receive messages defined in the Vertica protocol.
+This is used by the [vertica-nodejs](https://github.com/vertica/vertica-nodejs) client driver and is *not* meant to be
+used by developers writing their own applications with vertica-nodejs.
 
 ## License
```
Comment (jeremyprime) [inline packages/v-protocol/README.md]: The license under the top-level README has been changed to Apache 2.0, should we do the same for this (and the other packages)?

Comment (DMickens) [reply packages/v-protocol/README.md]: +1

**Review by DMickens — COMMENTED** [review]
Can we add a 'documentation' section similar to what is being added to the vertica-nodejs readme pointing out that this is a dependency of vertica-nodejs and point developers towards that package?

File: packages/v-protocol/README.md (hunk 1 above)
Comment (williammatherjones) [reply packages/v-protocol/README.md]: Looks like we still have the MIT license in each of the readme.md files in each package (v-connection-string, v-pool, v-protocol and vertica-nodejs).   Will you be changing that Danny?

Comment (DMickens) [reply packages/v-protocol/README.md]: I will indeed. vertica-nodejs has already been changed in the open PR

**Review by DMickens — APPROVED** [review]

**Pull request merged**